packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -173,6 +180,8 @@ export const ChatMessage: ComponentWithAs<'div', ChatMessageProps> &
     variables,
     header,
     details,
+    readStatus,
```

Comment:
Why isn't this just a `title` in the `readStatusIndicator` shorthand? I don't think we ever expose props from some shorthands as first class properties of the component.

Reply:
Removed